recipes/cyflann/meta.yaml

```diff
@@ -0,0 +1,58 @@
+{% set name = "cyflann" %}
+{% set version = "0.2.3" %}
+{% set sha256 = "a38cfdbaf7b4005f1f6ba417186f485483f78219bec5aa95f479f38551d234ae" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  fn: {{ name }}-{{ version }}.tar.gz
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: {{ sha256 }}
+
+build:
+  number: 0
+  script: python setup.py install --single-version-externally-managed --record record.txt
+  skip:
+    # Windows should work, but flann recipe isn't built there yet.
+    - True  # [win]
+
+requirements:
+  build:
+    - python
+    - setuptools
+    - toolchain
+    - numpy x.x
+    - flann >=1.9.1,<1.10
+    - cython
+  run:
+    - python
+    - numpy x.x
+    - flann >=1.9.1,<1.10
```

Comment:
Sure. My [shogun PR](https://github.com/conda-forge/staged-recipes/pull/3237) also needed to pin a bunch of other libraries: arpack, curl, glpk, json-c, lzo, protobuf, snappy (which is why I sent version bump PRs to a bunch of those today). Should I add those too?